04/app.js

```diff
@@ -0,0 +1,27 @@
+function handleClick(event) {
+    event.stopPropagation();
+
+    const targetDiv = event.currentTarget;
+
+    const delay = parseInt(targetDiv.dataset.time, 10);
+
+    setTimeout(() => {
+        targetDiv.classList.add('clicked');
+    }, delay);
+}
+
+function handleBodyClick(event) {
+
+    if (event.target === document.body) {
+        document.querySelectorAll('div.clicked').forEach(div => {
```

Comment:
Zwróć uwagę, że div-y już wyszukała wcześniej, może da się to jakoś wykorzystać?